all-spark-notebook/Dockerfile

```diff
@@ -20,11 +20,11 @@ RUN apt-get update && \
 USER $NB_USER
 
 # R packages
-RUN conda config --add channels r && \
+RUN conda config --add channels rdonnellyr && \
```

Comment:
I'd prefer to hold off until there's an official R release. Looking at the conda-recipe issue, it's still open for testing.